code/modules/mob_spawn/ghost_roles/golem_roles.dm

```diff
@@ -0,0 +1,96 @@
+
+//reserved file just for golems since they're such a big thing, available on lavaland and from the station
+
+//Golem shells: Spawns in Free Golem ships in lavaland. Ghosts become mineral golems and are advised to spread personal freedom.
+/obj/effect/mob_spawn/ghost_role/human/golem
+	name = "inert free golem shell"
+	desc = "A humanoid shape, empty, lifeless, and full of potential."
+	prompt_name = "free golem"
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "construct"
+	mob_species = /datum/species/golem
+	anchored = FALSE
+	move_resist = MOVE_FORCE_NORMAL
+	density = FALSE
+	var/has_owner = FALSE
+	var/can_transfer = TRUE //if golems can switch bodies to this new shell
+	var/mob/living/owner = null //golem's owner if it has one
+	you_are_text = "You are a Free Golem. Your family worships The Liberator."
+	flavour_text = "In his infinite and divine wisdom, he set your clan free to travel the stars with a single declaration: \"Yeah go do whatever.\""
+
+/obj/effect/mob_spawn/ghost_role/human/golem/Initialize(mapload, datum/species/golem/species = null, mob/creator = null)
+	if(species) //spawners list uses object name to register so this goes before ..()
+		name += " ([initial(species.prefix)])"
+		mob_species = species
+	. = ..()
+	var/area/A = get_area(src)
```

Comment:
:angry: (i can't tell if you added this or moved this so i won't mention the numerous others in this file)